go/vt/vtgate/evalengine/testcases/cases.go

```diff
@@ -1997,6 +1998,40 @@ func FnTimeToSec(yield Query) {
 	}
 }
 
+func FnToSeconds(yield Query) {
+	for _, t := range inputConversions {
+		yield(fmt.Sprintf("TO_SECONDS(%s)", t), nil)
+	}
+
+	timeInputs := []string{
+		`DATE'0000-00-00'`,
+		`0`,
+		`'0000-00-00'`,
+		`'00:00:00'`,
+		`DATE'2023-09-03 00:00:00'`,
+		`DATE'0000-00-00 00:00:00'`,
+		`950501`,
+		`'2007-10-07'`,
+		`'0000-01-01'`,
+		`TIME'00:00:00'`,
+		`TIME'120:01:12'`,
+	}
+
+	for _, t := range timeInputs {
+		yield(fmt.Sprintf("TO_SECONDS(%s)", t), nil)
+	}
+
+	mysqlDocSamples := []string{
+		`TO_SECONDS(950501)`,
+		`TO_SECONDS('2009-11-29')`,
+		`TO_SECONDS('2009-11-29 13:43:32')`,
```

Comment:
I don't think it would be a good idea to add `TO_SECONDS( NOW() )` here (although it's present in the docs) due to time difference between query execution in EvalEngine and MySQL.

Reply:
Agreed.